ledger-service/cli-opts/src/main/scala/cliopts/Metrics.scala

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.cliopts
+
+import com.daml.metrics.MetricsReporter
+import scopt.OptionDef
+
+import scala.concurrent.duration.FiniteDuration
+
+object Metrics {
+
+  def metricsReporterParse[C](parser: scopt.OptionParser[C])(
+      metricsReporter: Setter[C, Option[MetricsReporter]],
+      metricsReportingInterval: Setter[C, FiniteDuration],
+      hide: Boolean = false,
```

Comment:
Consider removing this flag and the accompanying logic as this is never called with `hide = false`.

Reply:
This logic exists to hide the option in the http json api.

Reply:
Thanks for the clarification, I seem to have missed that.